.github/workflows/ci.yml

```diff
@@ -30,5 +30,4 @@ jobs:
         run: bash <(curl -s https://codecov.io/bash) -t ${{secrets.CODECOV_TOKEN}} -B ${{ github.ref }} -f coverage/coverage-final.json
       # test the minimum supported peer dependency version
       - run: npm install typescript@4.1 rollup@3.0
-      # aka `npm test` without the `pretest/build`
-      - run: node .build/tests/index.js
+      - run: npm test
```

Comment:
I’m surprised this does not fail, skipping the build step was necessary because building with an outdated TS version used to fail.

Reply:
This PR removes the build step, so this is running the TS test files as if they're plain JS by stripping out the types using esbuild.